mkt/lookup/views.py

```diff
@@ -315,6 +318,18 @@ def app_summary(request, addon_id):
 
 
 @login_required
+@permission_required([('WebsiteLookup', 'View')])
+def website_summary(request, addon_id):
+    website = get_object_or_404(Website, pk=addon_id)
+    if not hasattr(website, 'keywords_list'):
```

Comment:
mmm, it's unfortunate that you have to do this, since we do this already in the serializer.

We should use attach_tags as a transformer in a custom manager method like `Website.objects.with_tags()` and use that method in this view and the API view... Maybe worth filing a code quality bug for that